packages/page-staking/src/Actions/useInactives.ts

```diff
@@ -20,28 +20,50 @@ interface Inactives {
   nomsWaiting?: string[];
 }
 
-function extractState (api: ApiPromise, stashId: string, slashes: Option<SlashingSpans>[], nominees: string[], { activeEra }: DeriveSessionIndexes, submittedIn: EraIndex, exposures: Exposure[]): Inactives {
-  const max = api.consts.staking?.maxNominatorRewardedPerValidator as u32;
+interface ExtractStateParams {
+  api: ApiPromise;
+  stashId: string;
+  slashes: Option<SlashingSpans>[];
+  nominees: string[];
+  activeEra: DeriveSessionIndexes['activeEra'] | undefined;
```

Comment:
Probably better to set this to `EraIndex` instead of `DeriveSessionIndexes['activeEra']` as it will be more explicit.